manager/manager.go

```diff
@@ -311,6 +346,8 @@ func (m *Manager) newCommandOutput(result *ssm.GetCommandInvocationOutput, err e
 	case "Success":
 		out.Output = aws.StringValue(result.StandardOutputContent)
 		out.OutputUrl = aws.StringValue(result.StandardOutputUrl)
+		out.StdErr = aws.StringValue(result.StandardErrorContent)
```

Comment:
I'm a bit torn on this one; we are already outputting stderr in the [Failed case](https://github.com/chrisdpa-tvx/ssm-sh/blob/master/manager/manager.go#L355-L361), so presumably if the command had a non-zero exit status we will get the stderr output. 

Do you have a specific use case where you need stderr on non-zero exit statuses? At any rate, if we want to output stderr on success, I think we should just combine the "Success" and "Failed" cases and always output both stdout and stderr?

Reply:
I found in windows the return code and stderr are not always set correctly. For example some command return 0 as a success but will still write to stderr

Reply:
For example
```» get-service bruce

i-053c236e196277a43 - Success:

get-service : Cannot find any service with service name 'bruce'.```

Reply:
Let's go with combining the switch cases for success and failure then, and always output both stdout and stderr. In cases where applications output this 'correctly', it'll just be one or the other anyway 👍